### PR TITLE
fix: Hide tag & relationship editors on Territory sidebar

### LINKED ISSUE #503: Let's hide All tags and All relations in the Territory sidebar
**Describe the idea**
Currently, the `All tags` and `All relations` dropdowns which show when you create or edit a feature are only useful for debugging purposes, and are mostly just confusing to end users. Let's set these to `display: none;` so that they are not showing but can be reactivated by devs for debugging.

**What's your role? Where and how do you currently use the application?**
Programs team at Dd

**Additional context**
Discussed with Gregor and Aliya today during a call.

### PR DESCRIPTION
**Pull request by rudokemper**
### First time contributor checklist:

- [X] I have read the [README](https://github.com/digidem/mapeo-desktop/blob/master/README.md) 

### Contributor checklist:

- [X] My contribution is **not** related to translations. _Please submit translation changes via our [Mapeo Crowdin project](https://crowdin.com/project/mapeo-desktop)_
- [X] My commits are in nice logical chunks with [good commit messages](http://chris.beams.io/posts/git-commit/)
- [ ] I have walked through the [QA Manual Testing Script](https://github.com/digidem/mapeo-desktop/blob/master/docs/QA.md) and updated it if necessary.
- [ ] If my changes depend upon an update to a dependency, I have updated the package-lock.json file using `npm install --package-lock`
- [ ] My changes have been tested with the [mobile app](https://github.com/digidem/mapeo-mobile/releases). 
- [X] My changes are ready to be shipped to users on Windows, Mac, and Linux using the production version of the application built with electron-builder.

### Description

Fixes #503.

This is just as much an experiment in doing open-source contributions to Mapeo code as it is fixing a very small thing discussed today with @gmaclennan - hiding several edit options in Mapeo Territory view that we are deeming to not be very helpful to show to end users.

Please let me know if I did this right in terms of filing an issue, submitting a PR, and versioning (I thought that `fix` was an appropriate prefix considering it's so small, but maybe it's a `feat` since it's not technically a bug?) I'd be happy to work with the team to improve on our guidelines for open-source contributions where helpful and needed 🙂

### CODE REVIEW

File: src/renderer/components/MapEditor/index.js [hunk 1]
```diff
@@ -96,6 +96,15 @@ insertCss(`
...
   }
+  .id-container .inspector-body .raw-tag-editor {
+    display: none;
+  }
+  .id-container .inspector-body .raw-member-editor {
+    display: none;
+  }
+  .id-container .inspector-body .raw-membership-editor {
+    display: none;
+  }
```
Comment (achou11) [inline src/renderer/components/MapEditor/index.js:107]: optional nitpick, but can shorten this to

```suggestion
  .id-container .inspector-body .raw-tag-editor,
  .id-container .inspector-body .raw-member-editor,
  .id-container .inspector-body .raw-membership-editor {
    display: none;
  }
```

**Review by achou11 — APPROVED** [review]
lgtm, thanks!

File: src/renderer/components/MapEditor/index.js (hunk 1 above)
Comment (gmaclennan) [reply src/renderer/components/MapEditor/index.js:107]: I personally prefer the more explicit option, it's easier to read.

**Review by gmaclennan — COMMENTED** [review]
@rudokemper pending approval from the programs team. If they agree please comment here and we will merge this change.

**Comment by rudokemper** [comment]
Discussed this with Programs team: there is full consensus on this, so will merge. Thanks!
